src/main/resources/assets/appliedenergistics2/lang/de_de.lang

```diff
@@ -429,11 +485,13 @@ item.appliedenergistics2.material.sky_dust.name=Himmelssteinstaub
 item.appliedenergistics2.material.card_crafting.name=Fertigungskarte
 
 item.appliedenergistics2.multi_part.annihilation_plane.name=ME-Annihilationsfeld
+item.appliedenergistics2.multi_part.fluid_annihilation_plane.name=ME-Flüssigkeiten-Annihilationsfeld
 item.appliedenergistics2.multi_part.identity_annihilation_plane.name=ME-Identitätsannihilationsfelds
 item.appliedenergistics2.multi_part.cable_anchor.name=Kabelanker
 item.appliedenergistics2.multi_part.cable_covered.name=Verdecktes ME-Kabel
 item.appliedenergistics2.multi_part.cable_glass.name=ME-Glaskabel
 item.appliedenergistics2.multi_part.cable_smart.name=Schlaues ME-Kabel
+item.appliedenergistics2.multi_part.cable_dense_covered.name=Verstecktes dichtes ME-Kabel
```

Comment:
With "dichtes" I relate something like a tight valve, better would be "verdichtetes".

Reply:
```suggestion
item.appliedenergistics2.multi_part.cable_dense_covered.name=Verstecktes verdichtetes ME-Kabel
```